demo/demo.html

```diff
@@ -3,8 +3,8 @@
   <head>
     <title>Demo of Logic and View Integration</title>
     <script src="../dist/components.js"></script>
-    <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script>
-    <!-- <script src="./temp/sfx-all-plugins-bundle.js"></script> -->
+    <!-- <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script> -->
+    <script src="./temp/sfx-all-plugins-bundle.js"></script>
```

Comment:
I've been trying to leave this commented out. However, given that the CDN version is pretty old, there is some argument to be made that we should switch to the temp bundle for now until we have a proper release cycle.

Reply:
Yes my tests will fail, since no terms or products will populate when using the cdn now. What would you like to do? @DanielMoniz-GBI

Reply:
I'm good with leaving this as you have it!